src/kernel/chainparams.h

```diff
@@ -109,10 +109,12 @@ class CChainParams
     uint64_t PruneAfterHeight() const { return nPruneAfterHeight; }
     /** Minimum free space (in GB) needed for data directory */
     uint64_t AssumedBlockchainSize() const { return m_assumed_blockchain_size; }
-    /** Minimum free space (in GB) needed for data directory when pruned; Does not include prune target*/
+    /** Minimum free space (in GB) needed for data directory when pruned; Does not include prune target */
```

Comment:
fa0ed7796fb9553c0458c7bc534f73441a991b0d: nit, don't touch unrelated things (may need to adjust your editor to not automatically do that).

Reply:
This was deliberate. The rest of the file has comments in the form of ```/** Alice Bob Carol */``` and this line didn't since I was working in the area I decided to make it consistent with the rest of the file.